src/main/java/codeu/model/data/Media.java

```diff
@@ -25,14 +26,16 @@
    * @param creation the creation time of this Media
    * @param content the content of this media
```

Comment:
1) Update Javadoc due to new parameter being added.
2) The new parameter conversationId can be null right? Because it can be picture on the profile page in which case there is no associated conversation?